cli/azd/pkg/azure/arm_template.go

```diff
@@ -20,6 +28,22 @@ type ArmTemplate struct {
 	Outputs        ArmTemplateOutputs              `json:"outputs"`
 }
 
+func (t ArmTemplate) TargetScope() DeploymentScope {
+	u, err := url.Parse(t.Schema)
+	if err != nil {
+		panic(fmt.Sprintf("bad schema for template: %s", t.Schema))
+	}
+
+	switch path.Base(u.Path) {
+	case "subscriptionDeploymentTemplate.json":
```

Comment:
Can we do case-insensitive matching here, unless `bicep` schema informs otherwise? I'm worried about `bicep`'s output -- we've ran into case sensitivity issues before

Reply:
Made the change here.